src/themes/semantic/bootstrap-table-semantic.js

```diff
@@ -43,7 +43,12 @@ $.BootstrapTable = class extends $.BootstrapTable {
 
   initPagination () {
     super.initPagination()
-    if (this.options.pagination && !this.options.onlyInfoPagination) {
+    let paginationParts = this.options.paginationParts
+    if (typeof paginationParts === 'string') {
+      paginationParts = paginationParts.replace(/\[|\]| /g, '').toLowerCase().split(',')
+    }
```

Comment:
> where is the advantage to save it to this.paginationParts instead to the variable

We can use `this.paginationParts` in every theme instead of repeat it.